sdk/storage/Azure.Storage.Common/src/StreamPartitioner.cs

```diff
@@ -26,7 +26,7 @@ public StreamPartitioner(Stream stream, MemoryPool<byte> memoryPool = default)
         /// Generate a forward-only sequence of substreams based on bufferSize.
         /// </summary>
         /// <returns>StreamPartition</returns>
-        public async Task<StreamPartition> ReadAsync(int size = Constants.DefaultBufferSize, CancellationToken ct = default)
+        public async Task<StreamPartition> ReadAsync(int size = Constants.DefaultBufferSize, bool async = true, CancellationToken ct = default)
         {
             // TODO these operations should be simplified with Memory- and Span-accepting APIs in future NET Standard
```

Comment:
Is this doable now, without losing netfx 4.6.x?

Reply:
Yes, `System.Memory` (https://www.nuget.org/packages/System.Memory/) supports net46*